utils/src/main/kotlin/OrtProxySelector.kt

```diff
@@ -105,22 +105,26 @@ class OrtProxySelector(private val fallback: ProxySelector? = null) : ProxySelec
      * Add a [proxy with optional password authentication][authenticatedProxy] that can handle the [protocol]. The
      * [origin] is a string that helps to identify where a proxy definition comes from.
      */
-    fun addProxy(origin: String, protocol: String, authenticatedProxy: AuthenticatedProxy) {
+    fun addProxy(origin: String, protocol: String, authenticatedProxy: AuthenticatedProxy): OrtProxySelector {
         val (proxy, authentication) = authenticatedProxy
         proxyAuthentication[proxy] = authentication
 
         val proxiesForOrigin = proxyOrigins.getOrPut(origin) { mutableMapOf() }
         val proxiesForProtocol = proxiesForOrigin.getOrPut(protocol) { mutableListOf() }
         proxiesForProtocol += proxy
+
+        return this
```

Comment:
Why not use `apply` like in the function below?

Reply:
Simply adding a line without changing indentation seemed less intrusive to me here. Doing the same was not easily possible in the other function.

Reply:
But the result would be more consistent.

Reply:
Actually, this is the kind of variety I personally enjoy in a code base. Anyway, I don't insist on it, and have changed it.